### ISSUE TITLE
Where can I find sikulixapi 1.1.4 final in Maven?

### ISSUE THREAD

**Issue by DavidPerezIngeniero** [RaiMan/SikuliX1#50]
The documentation tells:

> The repository URL:http://oss.sonatype.org/content/groups/public
> The coordinates are:
> <groupId>com.sikulix</groupId>
> <artifactId>sikulixapi</artifactId>
> <version>1.1.4-SNAPSHOT</version>

But where can I find 1.1.4 final?
As you can see, in [Maven central](https://mvnrepository.com/artifact/com.sikulix/sikulixapi), there is only till version 1.1.2.

If I try to use 1.1.4-SNAPSHOT:

`checksum` format error: ~/.cache/coursier/v1/http/oss.sonatype.org/content/groups/public/com/sikulix/sikulixapi/1.1.4-SNAPSHOT/sikulixapi-1.1.4-SNAPSHOT.pom.sha1

**Comment by DavidPerezIngeniero** [comment]
OK, thanks for the answer.
And what about 1.1.3 that has the same problem?

**Comment by RaiMan** [comment]
Same as 4 1.1.4

**Comment by RaiMan** [comment]
You are right😎
I have to revise all this when I am back from vacation☀️☀️☀️

**Comment by MichielYKamp** [comment]
I'm hoping you will be able to check this out soon. The source http://oss.sonatype.org/content/groups/public does't seem to exist.

I would be happy with version 1.1.3 also.

**Comment by RaiMan** [comment]
... and this snapshot repo setting (as mentioned in the docs) works also:
```
    <repository>
      <!--OSSRH: com.sikulix-->
      <id>com.sikulix</id>
      <name>com.sikulix</name>
      <url>https://oss.sonatype.org/content/groups/public</url>
      <layout>default</layout>
      <snapshots>
        <enabled>true</enabled>
        <updatePolicy>always</updatePolicy>
      </snapshots>
    </repository>
```

**Comment by RaiMan** [comment]
... and this is the current state of relevant versions (with respect to Maven Central)

**1.1.4** 
OSSRH snapshot only

**1.1.3** 
OSSRH snapshot only (but not changed anymore, hence a "stable" snapshot ;-)

**1.1.2** 
on Maven Central

Due to the effort to spend, there will not be a 1.1.3 on Maven Central. 
You have to live with the "stable" snapshot.

... and a final 1.1.4 on Maven Central will not exist before April next year.

**Comment by DavidPerezIngeniero** [comment]
Thanks RaiMan!
This is an improvement

**Comment by pguermo** [comment]
Hello RaiMan,
Releasing a xxxx-SNAPSHOT version for a final version is a problem for the nexus repositories/proxies. The released 1.1.3-SNAPSHOT is otherwise not usable behind a nexus "Release" proxy. 
Could you release a final 1.1.3.1 based on the 1.1.3 ? 
Thanks in advance.

**Comment by RaiMan** [comment]
@pguermo 
> released 1.1.3-SNAPSHOT

... is not a release in Maven meaning. It is a snapshot, that will most probably (or let's say surely) never be modified again.

> release a final 1.1.3.1

... as mentioned: there will not be a 1.1.3-whatever on MavenCentral.

So you either have to live with that or find your own solution.

The next 3 months I have some private priorities and my spare time will be dedicated to making 1.1.4 stable for a final in April next year.

Sorry for any inconveniences.
